### ISSUE TITLE
[Issue]: Collections delete prompt is misleading

### ISSUE THREAD

**Issue by cinderblockgames** [jellyfin/jellyfin-web#5040]
### Please describe your bug

If you click Delete on a Collection, you get the following prompt:

```
Deleting this item will delete it from both the file system and your media library. Are you sure you wish to continue?
```

Collections don't exist on the file system, and it gives me a pause every time, like "am I about to delete my movies by accident?"

### Jellyfin Version

10.8.0

### Code of Conduct

- [x] I agree to follow this project's Code of Conduct

**Comment by alexfornuto** [comment]
It looks like this uses the value from the key `ConfirmDeleteItem`, which is used by `deleteHelper.js`, which is used by  `itemContextMenu.js`, which is used by... you get the idea.

I wholeheartedly agree with this improvement and wanted to submit a patch myself, but it's over my non-developer head to do. I reckon there would either need to be a new `itemContextMenu` specific for collections, or some sort of catch for when it's used in the Collections page.

**Comment by alexfornuto** [comment]
Non-stale update. The problem has been identified, and solutions proposed, it just needs a brave soul to make it happen.

**Comment by ibizaman** [comment]
Not stale just yet!

**Comment by kevin201** [comment]
Agree. It's a very misleading wording for a warning message. It's the equivalent of deleting a media library, and should be treated as such, IMHO.

**Comment by lichenophile** [comment]
Wild that this issues so old haha

> Deleting this collection will delete it from your media library as well as the collection folder on the file system and all its content. The original media files will be left untouched. Are you sure you wish to continue?

Can anyone clarify for me? I now understand from this that the `Delete` option under the collection `...` menu does not delete my media, but I want to confirm that it also does not remove media from my pre-existing libraries. (IE it restores my library view to the state it was before the Collection 'grouping')

Logically, I want to assume deleting a collection does nothing but delete the grouping and is entirely safe to do but boy it'd be a pain if I had to readd things to my libraries and lost playback stats, metadata etc

(I created a massive collection for anime series + anime movies a while back, have been scared to delete it but want to remove it as it hides movies from the library view, they're all grouped under the collection instead)

**Comment by NoricSteel** [comment]
> Wild that this issues so old haha
> 
> Can anyone clarify for me? I now understand from this that the `Delete` option under the collection `...` menu does not delete my media, but I want to confirm that it also does not remove media from my pre-existing libraries. (IE it restores my library view to the state it was before the Collection 'grouping')
> 
> Logically, I want to assume deleting a collection does nothing but delete the grouping and is entirely safe to do but boy it'd be a pain if I had to readd things to my libraries and lost playback stats, metadata etc

Had the same issue. I can confirm. I deleted one of my collections, the movies are still there, just the grouping was deleted.

**Comment by Mondriaan123** [comment]
Very unclear, I have two collections: "comedies, animation" and "comedies." I want to get rid of the  "comedies, animation" Please help!

**Comment by githuemphasis129** [comment]
hi.  it seems this is still relevant.  i have a misidentified collection and the underlying media is there but not being found.  seems like something is just wrong with this one collection.  all the others are great.  so i was going to delete it but the warning message is too misleading imo.  so i didn't delete the collection even though it seems like the right thing to do bc i afraid i'll delete media.  

it would be helpful to clarify what specifically will be deleted.  just the collection?  the underlying media?

very appreciative for jellyfin though so this is not a complaint.  thanks you devs.


svr: 10.11.0

build version: 2025100605

**Comment by somu1795** [comment]
why this issue is not being patched? The warning statement is pretty misleading. This should have been fixed a long time ago!

**Comment by alexfornuto** [comment]
Thanks @dfederm!

**Comment by LarsSimonsen** [comment]
The prompt is now "Deleting this item will delete it from both the file system and your media library. Are you sure you wish to continue?" So, if this was ever fixed, there seems to be a regression.

**Comment by alexfornuto** [comment]
Confirmed that the behavior from https://github.com/jellyfin/jellyfin-web/commit/8b30b5b6c8f48eb79546a4a27f1f9d317190f818 is no longer present. Tagging @thornbill as I cannot myself find any regression in the files he updated, so it might be elsewhere in the code base.

**Comment by viown** [comment]
The changes in the merged PR will be available in 10.12.
